miden/src/cli/verify.rs

```diff
@@ -62,4 +69,26 @@ impl VerifyCmd {
 
         Ok(())
     }
-}
+
+    pub fn infer_defaults(&self)->(PathBuf,PathBuf){
```

Comment:
As mentioned in another comment, I'd probably infer input/output files based on the proof file rather than the other way around.

Also, I don't think this function needs to be public, right?